src/components/ToolMenu/Draw/ColorPickDrawer/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React,
+{
+  useState
+} from 'react';
+
+import {
+  faPaintBrush
+} from '@fortawesome/free-solid-svg-icons';
+
+import {
+  FontAwesomeIcon
+} from '@fortawesome/react-fontawesome';
+
+import {
+  Button,
+  Drawer,
+  DrawerProps
+} from 'antd';
+
+import {
+  useTranslation
+} from 'react-i18next';
+
+import {
+  ClientPluginComponentProps
+} from '../../../../plugin';
+
+import './index.less';
+import ColorPicker from './ColorPicker';
+
+export type ColorPickDrawerProps = ClientPluginComponentProps & DrawerProps & {};
```

Comment:
Please remove the `ClientPluginComponentProps` since it's not a plugin component.